Rectangle/Base.lproj/Main.storyboard

```diff
@@ -3150,55 +3150,71 @@
+                                        <rect key="frame" x="0.0" y="165" width="450" height="20"/>
+                                        <subviews>
+                                            <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="5An-8B-vsH">
+                                                <rect key="frame" x="-2" y="4" width="132" height="16"/>
+                                                <textFieldCell key="cell" lineBreakMode="clipping" title="Repeated commands" id="2Zm-fl-PcC">
+                                                    <font key="font" metaFont="system"/>
+                                                    <color key="textColor" name="labelColor" catalog="System" colorSpace="catalog"/>
+                                                    <color key="backgroundColor" name="textBackgroundColor" catalog="System" colorSpace="catalog"/>
+                                                </textFieldCell>
+                                            </textField>
+                                            <popUpButton verticalHuggingPriority="750" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="pVH-s3-FHn">
+                                                <rect key="frame" x="133" y="-4" width="321" height="25"/>
+                                                <popUpButtonCell key="cell" type="push" title="do nothing" bezelStyle="rounded" alignment="left" lineBreakMode="truncatingTail" state="on" borderStyle="borderAndBezel" tag="2" imageScaling="proportionallyDown" inset="2" selectedItem="jww-Ju-S3d" id="ccx-Gx-MGO">
+                                                    <behavior key="behavior" lightByBackground="YES" lightByGray="YES"/>
+                                                    <font key="font" metaFont="menu"/>
+                                                    <menu key="menu" id="TkU-PT-5p8">
+                                                        <items>
+                                                            <menuItem title="do nothing" state="on" tag="2" id="jww-Ju-S3d"/>
+                                                            <menuItem title="cycle through displays" tag="4" id="XlM-ch-cLG"/>
+                                                            <menuItem title="cycle ½, ⅔, and ⅓ on half actions" id="gHH-BV-5kP"/>
+                                                            <menuItem title="move to adjacent display on left or right" tag="1" id="Z9d-Rl-RVq"/>
+                                                            <menuItem title="move to adjacent on left/right, or cycle size on half" tag="3" id="3GE-la-fAZ"/>
```

Comment:
Note that the tag property of these menu items indicates the value of the `SubsequentExecutionMode`